packages/website/pages/terms.js

```diff
@@ -39,59 +41,58 @@ export default function TermsOfService() {
           If you do not agree to all the terms and conditions of this agreement,
           then you may not access the Website or use any services.
         </p>
-        <h2 className="chicagoflf">
-          <HashLink id="storage-term">Storage Term</HashLink>
-        </h2>
-        <p className="lh-copy">
-          Data will be stored at no cost to the user on IPFS for as long as
-          Protocol Labs, Inc. continues to offer free storage for NFT’s. Data
-          will continue to be persisted ad infinitum <strong>or</strong> until
-          Protocol Labs decides to conclude the NFT.storage project. Prior to
-          termination, Protocol Labs will provide 90 days notice to users via
-          email to allow users enough time to make arrangements for storing
-          their data by other means.
-        </p>
 
         <h2 className="chicagoflf">
           <HashLink id="data-limits">Data Limits</HashLink>
         </h2>
         <p className="lh-copy">
-          The service accepts arbitrary upload sizes up to 30GB per upload.
-          Currently, the service has no data upload limit. Protocol Labs may
-          amend this term limit at its sole discretion.
+          The Service is offered for the creation and storage of NFTs. Use of the 
+          Service to store other types of data is not permitted. The Service 
+          accepts arbitrary upload sizes up to 31GiB per upload. Currently, the 
+          Service does not limit users in the total data they can upload. Protocol 
```

Comment:
Suggestion: "...does not limit users with respect to the total amount of data they can upload."